packages/gateway/src/durable-objects/summary-metrics.js

```diff
@@ -35,13 +49,22 @@ const TOTAL_CACHED_CONTENT_LENGTH_BYTES_ID = 'totalCachedContentLengthBytes'
 const CONTENT_LENGTH_HISTOGRAM_ID = 'contentLengthHistogram'
 // Key to track response time histogram
 const RESPONSE_TIME_HISTOGRAM_ID = 'responseTimeHistogram'
+// Key to track response time histogram by pin status
+const RESPONSE_TIME_HISTOGRAM_BY_PIN_STATUS_ID =
+  'responseTimeHistogramByPinStatus'
+// Key to track responses by content status
+const TOTAL_RESPONSES_BY_CONTENT_STATUS_ID = 'totalResponsesByContentStatus'
+// Key to track responses by pin status
+const TOTAL_RESPONSES_BY_PIN_STATUS_ID = 'totalResponsesByPinStatus'
 
 /**
  * Durable Object for keeping summary metrics of gateway.nft.storage
  */
 export class SummaryMetrics0 {
   constructor(state) {
     this.state = state
+    // @ts-ignore we don't need token just for check
+    this.nftStorageClient = new NFTStorage({})
```

Comment:
For just check, we don't need a token, while it is "required" in NFTstorage (but param called option)

@hugomrdias should we make it optional in the types?

Reply:
yes we should, but here if you just use it for check call might be better to not include the client.

Reply:
you mean,  just HTTP the endpoint?

Reply:
yep, I agree, if we just want to hit the public check endpoint, maybe just add a helper function around a fetch